.travis.yml

```diff
@@ -8,4 +8,4 @@ env:
   - DIR=brunch
   - DIR=webpack
 script:
-  - ./travis-runner.sh
+  - bash ./travis-runner.sh
```

Comment:
without this travis don't wanna run bash script.
You can take a look at this [build](https://travis-ci.org/marionettejs/marionette-integrations/jobs/133975023#L157)
I thought that problem is in absent lines in `travis.yaml`
```
env:
  - secure: secure-key
```
but it appears that adding `bash` fixing this problem.
Maybe I'm wrong but I didn't find another way how to make it works. Any suggestions?